content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -206,72 +147,129 @@ of when a :guilabel:`URL Regex` action is triggered (e.g., a page visit).
-     was received when the session ended.
-   - The :guilabel:`% Happy` smart button includes a log of ratings that were left by live chat
-     attendees, and are labeled by date, time, and the support agent who was responsible for the
-     live chat session.
-
-Add the URL for the pages this channel will be applied to in the :guilabel:`URL Regex` field. If
-this channel will only be available to users in specific countries, add them to the
+   If a chatbot is added to a live chat channel, a new :guilabel:`Chatbots` smart button appears on
+   the channel settings form. Click here to create and update the chatbot *script*. Each line in the
+   script contains a :guilabel:`Message`, :guilabel:`Step Type`, :guilabel:`Answers`, and
+   conditional *Only If* logic that applies when certain pre-filled answers are chosen. To create
+   more steps in the script, click :guilabel:`Add a line` and fill out the script steps form
+   according to the desired logic.
+
+Add the URLs for the pages where the channel should appear to in the :guilabel:`URL Regex` field. If
+this channel should only be available to users in specific countries, add them to the
 :guilabel:`Country` field. If this field is left blank, the channel will be available to all site
 visitors.
 
 .. image:: receiving_tickets/receiving-tickets-channel-rules.png
    :align: center
-   :alt: View of the kanban cards for the available Live Chat channels.
+   :alt: View of the Kanban cards for the available Live Chat channels.
 
 Use the live chat widget
 ************************
 
-The :guilabel:`Widget` tab on the live chat channel form offers an embeddable website widget, or a
-shortcode for instant customer/supplier access to a live chat window.
+The :guilabel:`Widget` tab on the live chat channel form offers a website widget that can be added
+to third party platforms. Additionally, a short code is available, that can provide instant access
+to a live chat window.
 
 The live chat :guilabel:`Widget` can be applied to websites created through Odoo by navigating to
-the :menuselection:`Website --> Configuration --> Settings`. Then scroll to the :guilabel:`Live
-Chat` section, and select the channel to add to the site. Click :guilabel:`Save` to apply.
+the :menuselection:`Website app --> Configuration --> Settings --> Email & Marketing`. Then scroll
+to the :guilabel:`Live Chat` section, and select the channel to add to the site. Click
+:guilabel:`Save` to apply.
 
-To add the widget to a website created on a third-party platform, click :guilabel:`COPY` and paste
+To add the widget to a website created on a third-party platform, click :guilabel:`Copy` and paste
 the code into the `<head>` tag on the site.
 
-Likewise, to send a live chat session to a customer or supplier, click the second :guilabel:`COPY`
+Likewise, to send a live chat session to a customer or supplier, click the second :guilabel:`Copy`
 button which contains a link to join directly.
```

Comment:
You don't specify what code we are copying. There are two options, one is javascript and the other is an URL